core/app/[locale]/(default)/(auth)/register/_actions/register-customer.ts

```diff
@@ -84,7 +86,7 @@ export const registerCustomer = async ({ formData, reCaptchaToken }: RegisterCus
 
     return {
       status: 'error',
-      error: result.errors.map((error) => error.message).join('\n'),
+      error: createErrorsList(result.errors),
```

Comment:
Why don't we just change `error` to be `errors` and update the UI to render a list `ul` instead. It'll be better for accessibility instead of using a unicode character to fake it.

Reply:
Redone. Please check and if it looks appropriate I will extend to other account related forms